localstack/config.py

```diff
@@ -580,6 +580,9 @@ def in_docker():
 # whether lambdas should use stay open mode if executed in "docker-reuse" executor
 LAMBDA_STAY_OPEN_MODE = is_in_docker and is_env_not_false("LAMBDA_STAY_OPEN_MODE")
 
+# truncate output string slices value
+LAMBDA_TRUNCATE_STDOUT = int(os.getenv("LAMBDA_TRUNCATE_STDOUT") or 2000)
```

Comment:
In slack you mentioned 10000 as a sensible default value (except in CI), did you change your mind?

Reply:
I did! it would require tinkering with the CI pipeline, this keeps as it is and up to the user configure.